test/packages/benchmarks/rally_benchmark/_dev/benchmark/rally/logs-benchmark.yml

```diff
@@ -0,0 +1,20 @@
+---
+description: Benchmark 20000 events ingested
+input: filestream
```

Comment:
Is the input config needed?

Reply:
unluckily yes :(
I was hoping to be able to get rid of it, and hardcode in the code an input type that requires no vars.

the problem is that the package and policy will be installed, before running rally, and they must match an input defined by the package I'm going to benchmark, otherwise the policy will be rejected.

I'm trying to get rid of it, but it requires installing a common input for every package that's not part of the original package

Reply:
Is this the issue around "each package need requires to have a policy"? @tommyers-elastic Is this still required? How did you deal with it in the context of firehose? It seems https://github.com/elastic/integrations/pull/7146 does not contain an input? 

It is also not clear to me, why it is needed here? What exactly is triggered when installing the package? Do you create a policy or only install the package? I would assume for only installing the package, this is not needed?

Reply:
I looked at what happens during the rally run and it sets up a policy. I don't think this should happen, it should only install the package itself, no policy setup. Basically what `elastic-package install` does. This should also remove the requirement for an input.

Reply:
> I looked at what happens during the rally run and it sets up a policy. I don't think this should happen, it should only install the package itself, no policy setup. Basically what `elastic-package install` does. This should also remove the requirement for an input.

indeed! I've realized it as well and remove everything: I've just pushed the commit :)

Reply:
It seems the input config is still here? Is this now still needed?

Reply:
Running it again, it seems now it does not install the package at all anymore? We need the package for the pipeline and templates.

Reply:
sorry. there was some leftover.

about the package not installed, according to [fleet api](https://www.elastic.co/guide/en/fleet/current/fleet-api-docs.html) I don't see a way to install a package without creating a policy.
but then we are back to the beginning where the policy needs to match the inputs of the package.
I guess for firehose integration it works without input because the package has no input

Reply:
I think the API call you are looking for is here: https://www.elastic.co/guide/en/fleet/current/fleet-apis.html#installPackage

Any package can be just installed without policy, policy is just "another" asset.

Reply:
@jsoriano I wonder if we need to rely on the package registry at all here for installation or could instead us `elastic-package install` command so there is no dependency on the registry. Maybe that already happens?

Reply:
Yes, for versions of Kibana that support it, `elastic-package install` installs packages without using the registry, and without requiring a policy.

You can find here how the system test runner installs packages to leverage the best available method: https://github.com/elastic/elastic-package/blob/a7e19f9823f38bff7822a4bd3c61f5f61d4bb81a/internal/testrunner/runners/system/runner.go#L490